node/src/manager/commands/prune.rs

```diff
@@ -169,24 +188,38 @@ pub async fn run(
 
     println!("prune {deployment}");
     println!("    latest: {latest}");
-    println!("     final: {}", latest - ETH_ENV.reorg_threshold);
+    println!("     final: {}", latest - ENV_VARS.reorg_threshold);
     println!("  earliest: {}\n", latest - history);
 
+    let mut req = PruneRequest::new(
+        &deployment,
+        history,
+        ENV_VARS.reorg_threshold,
```

Comment:
A minor inconsistency is that this env value might be different between the index node and graphman, but improving this will require a general rethink of finality configuration.

Reply:
Yeah, the global `reorg_threshold` is something that I'd love for us to improve on. I feel that that is something that should be settable for each chain individually (e.g., in `graph-node.toml`)